Mage Pré-configuraçao de skills

```diff
@@ -0,0 +1,14 @@
+automacro magesSpells {
+	JobID = 2, 9 ,16
+	exclusive 1
+	run-once 1
+	#Mapas de base -> ein_fild09 | pay_fild02 | mjolnir_11 | mjolnir_01 | mjolnir_02 | cmd_fild01
+	call {
+		do conf attackSkillSlot_0 Relampago
+			do conf attackSkillSlot_0_monsters Esporo #Contra elemento agua
+		do conf attackSkillSlot_1 Lanças de Fogo
+			do conf attackSkillSlot_1_monsters Jiboía, Rabo de Verme, Caramelo, Pé Grande, Creamy, Percevejo, Poporing, Besouro-Chifre, Yoyo, Grove, Flora, Argiope, Argos, Porcellio, Metaling  #Contra Elemento Terra (de vento/veneno estou botando aqui tb porque nao tem dano elemental bom)
+		do conf attackSkillSlot_2 Lanças de Gelo
+			do conf attackSkillSlot_2_monsters Salgueiro Ancião #Contra Elemento Fogo	
```

Comment:
só tem um probleminha..

o opnkore nao cria um bloco novo de attackSkillSlot quando vc usa o comando do conf
mesmo que vc use com a flag -f (que força a ser criada) para blocos não funciona

por esse motivo rolou toda uma gambiarra pra fazer um bloco diferente , usando sub

Reply:
então ele teria que editar o único bloco dependendo do monstro,nossa isso dificulta bastante... vou ver o que da pra fazer, mas como a intenção seria adaptar para todas as classes isso é algo essencial para classes de mago uparem sem sofrimento

Reply:
posso te mostrar como fazer o sub, acho que vc consegue entender de boas
```perl
sub configurarPotLaranja_usar {
	open (my $fh, '>>:encoding(UTF-8)', Settings::getControlFilename('config.txt'));
	print $fh "useSelf_item Poção Laranja {\n";
	print $fh "	hp < 50%\n";
	print $fh "	disabled\n";
	print $fh "	label usarPot\n";
	print $fh "}\n";
	close($fh);
	Commands::run("reload config");
}
```

esse é pra configurar um novo bloco de useSelf_item, o código é o mesmo pra um attackSkilSlot
só mudar o que ta dentro das aspas mesmo.

esse eespaço que tem, é o tab